### DISCUSSION TITLE
how to get auth?

### DISCUSSION THREAD

**Discussion by gavenwan** [fastapi-users/fastapi-users#571] [Q&A] [answered]
1-  I use "login"module to get "access-token"

2- and I put  the token in the request header  to get user/me 
    Rquest header
   Content-Type: application/x-www-form-urlencoded;charset=UTF-8
  Authorization:    eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiYjI2NDUyYWQtMDZmNi00YjIzLWJkNzctODI5OGNmMWIxZDcwIiwiYXVkIjoiZmFzdGFwaS11c2VyczphdXRoIiwiZXhwIjoxNjE2NjQyODc1fQ.m86gQ3DDm5qVEwGtNnnXU6pTYZuuAaefybrgDw-03eY
  token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiYjI2NDUyYWQtMDZmNi00YjIzLWJkNzctODI5OGNmMWIxZDcwIiwiYXVkIjoiZmFzdGFwaS11c2VyczphdXRoIiwiZXhwIjoxNjE2NjQ0MTg5fQ.9aEYW5mOnxAjsFWEe-MNVQj5tNYvMnoddzVFBNu1Uxw
Connection: close

3- the response  is "detail": "Unauthorized" or    "detail": "Not Found"

what should I do to auth request?

**Comment by frankie567** [comment, marked as answer]
The header should be in the form `Authorization: Bearer YOURTOKENHERE`. Everything is explained here:
* https://frankie567.github.io/fastapi-users/usage/flow/#3-get-my-profile
* https://frankie567.github.io/fastapi-users/configuration/authentication/jwt/#authentication

**Reply by gavenwan** [reply]
thanks a lot. I forgot to put "Bearer" string before token string in the header ..
anyway, problem solved, thanks again
